src/main/java/uk/gov/justice/hmpps/prison/service/InmateService.java

```diff
@@ -148,17 +148,12 @@ public List<InmateBasicDetails> getBasicInmateDetailsForOffenders(final Set<Stri
             results.addAll(
                 repository.getBasicInmateDetailsForOffenders(new HashSet<>(offenderList), canViewAllOffenders, caseloads, active)
                     .stream()
-                    .map(offender -> {
-                            var middleName = "";
-                            if (offender.getMiddleName() != null) {
-                                middleName = WordUtils.capitalizeFully(offender.getMiddleName()).trim();
-                            }
-                            return offender.toBuilder()
+                    .map(offender ->
+                            offender.toBuilder()
                                 .firstName(WordUtils.capitalizeFully(offender.getFirstName()))
-                                .middleName(middleName)
```

Comment:
This was wrong anyway but didn't catch all repos returning middle names. Better to fix up the SQL everywhere it's wrong.